static/bidder-params/adf.json

```diff
@@ -8,7 +8,21 @@
       "type": ["integer", "string"],
       "pattern": "^\\d+$",
       "description": "An ID which identifies the placement selling the impression"
+    },
+    "inv": {
+      "type": ["integer"],
+      "description": "An ID which identifies the Adform inventory source id"
+    },
+    "mname": {
+      "type": ["string"],
+      "description": "A Name which identifies the placement selling the impression"
     }
   },
-  "required": ["mid"]
+  "anyOf":[
+    {
+      "required": ["mid"]
+    }, {
+      "required": ["inv", "mname"]
```

Comment:
When changes in the `required` field of the JSON shcema occur, we encourage bidders to make sure future requests are going to comply because, if not, Prebid Server it will mark the whole request as failed and not send a bid request to any bidder. So adding or modifying a required field can break the entire prebid setup of a publisher.